website/client/components/challenges/myChallenges.vue

```diff
@@ -16,7 +17,7 @@
           span(v-once) {{$t('createChallenge')}}
 
     .row
-      .no-challenges.text-center.col-md-6.offset-3(v-if='filteredChallenges.length === 0')
```

Comment:
why did you change this from checking `filteredChallenges.length ` to `challenges.length `?

Reply:
that makes sens, thanks @cvuorinen

Reply:
The design would look similar to the empty state. The messaging would need to be updated to something along the lines of "We couldn't find any Challenges that match."

Reply:
So should I go ahead and implement something based on that or what is the next course of action?

Or would it be ok to merge this PR first and then make a new one for that? I can work on it if that's okay with everyone, but would like to get this one out of the way since it's been hanging quite long already.

Reply:
I think this one is good to go! For the other issue let's make a new PR. I'll make sure to finally review this before we merge the next badge of PRs